packages/wallet/primitives/src/network.ts

```diff
@@ -1,630 +1,1229 @@
-  name: 'telos',
-  rpc: getRpcUrl('telos'),
-  chainId: 40n,
-  explorer: 'https://explorer.telos.net/network/',
-  nativeCurrency: {
-    name: 'TLOS',
-    symbol: 'TLOS',
-    decimals: 18,
-  },
-}
+  // LAOS
+  LAOS: 6283n,
+  LAOS_SIGMA_TESTNET: 62850n,
 
-export const TelosTestnet: Network = {
-  name: 'telos-testnet',
-  rpc: getRpcUrl('telos-testnet'),
-  chainId: 41n,
-  explorer: 'https://explorer-test.telos.net/network',
-  nativeCurrency: {
-    name: 'TLOS',
-    symbol: 'TLOS',
-    decimals: 18,
-  },
-}
+  // ETHERLINK
+  ETHERLINK: 42793n,
+  ETHERLINK_TESTNET: 128123n,
 
-export const B3: Network = {
-  name: 'b3',
-  rpc: getRpcUrl('b3'),
-  chainId: 8333n,
-  explorer: 'https://explorer.b3.fun/',
-  nativeCurrency: {
-    name: 'Ether',
-    symbol: 'ETH',
-    decimals: 18,
-  },
-}
+  // MOONBEAM
+  MOONBEAM: 1284n,
+  MOONBASE_ALPHA: 1287n,
 
-export const B3Sepolia: Network = {
-  name: 'b3-sepolia',
-  rpc: getRpcUrl('b3-sepolia'),
-  chainId: 1993n,
-  explorer: 'https://sepolia.explorer.b3.fun/',
-  nativeCurrency: {
-    name: 'Ether',
-    symbol: 'ETH',
-    decimals: 18,
-  },
-}
+  // MONAD
+  MONAD_TESTNET: 10143n,
 
-export const ApeChain: Network = {
-  name: 'apechain',
-  rpc: getRpcUrl('apechain'),
-  chainId: 33139n,
-  explorer: 'https://apechain.calderaexplorer.xyz/',
-  nativeCurrency: {
-    name: 'ApeCoin',
-    symbol: 'APE',
-    decimals: 18,
-  },
-}
+  // SOMNIA
+  SOMNIA_TESTNET: 50312n,
+  SOMNIA: 5031n,
 
-export const ApeChainTestnet: Network = {
-  name: 'apechain-testnet',
-  rpc: getRpcUrl('apechain-testnet'),
-  chainId: 33111n,
-  explorer: 'https://curtis.explorer.caldera.xyz/',
-  nativeCurrency: {
-    name: 'ApeCoin',
-    symbol: 'APE',
-    decimals: 18,
-  },
-}
+  // INCENTIV
+  INCENTIV_TESTNET: 11690n,
 
-export const Blast: Network = {
-  name: 'blast',
-  rpc: getRpcUrl('blast'),
-  chainId: 81457n,
-  explorer: 'https://blastscan.io/',
-  nativeCurrency: {
-    name: 'Ether',
-    symbol: 'ETH',
-    decimals: 18,
-  },
-}
+  // SEI
+  SEI: 1329n,
+  SEI_TESTNET: 1328n,
+} as const
 
-export const BlastSepolia: Network = {
-  name: 'blast-sepolia',
-  rpc: getRpcUrl('blast-sepolia'),
-  chainId: 168587773n,
-  explorer: 'https://sepolia.blastexplorer.io/',
-  nativeCurrency: {
-    name: 'Ether',
-    symbol: 'ETH',
-    decimals: 18,
-  },
-}
+export type ChainId = (typeof ChainId)[keyof typeof ChainId]
 
-export const SkaleNebula: Network = {
-  name: 'skale-nebula',
-  rpc: getRpcUrl('skale-nebula'),
-  chainId: 1482601649n,
-  explorer: 'https://green-giddy-denebola.explorer.mainnet.skalenodes.com/',
-  nativeCurrency: {
-    name: 'SKALE Fuel',
-    symbol: 'sFUEL',
-    decimals: 18,
+export const ALL: Network[] = [
+  {
+    chainId: ChainId.MAINNET,
+    type: NetworkType.MAINNET,
+    name: 'mainnet',
+    title: 'Ethereum',
+    rpc: getRpcUrl('mainnet'),
+    logoURI: `https://assets.sequence.info/images/networks/medium/${ChainId.MAINNET}.webp`,
```

Comment:
since all logos will use the same url maybe we should create a `getLogoUrl(chainId: ChainId) => ``https://assets.sequence.info/images/networks/medium/${chainId}.webp`` function

`logoUrl: getLogoUrl(ChainId.MAINNET)`